src/AlertsManagement/AlertsManagement/ActionRuleCommands/UpdateAzureActionRule.cs

```diff
@@ -0,0 +1,147 @@
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.AlertsManagement.OutputModels;
+using Microsoft.Azure.Management.AlertsManagement.Models;
+using Newtonsoft.Json;
+using System.Collections;
+using Microsoft.Azure.PowerShell.Cmdlets.AlertsManagement.Properties;
+
+namespace Microsoft.Azure.Commands.AlertsManagement
+{
+    [Cmdlet("Update", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ActionRule", DefaultParameterSetName = ByNameSimplifiedPatchParameterSet,
+        SupportsShouldProcess = true)]
+    [OutputType(typeof(PSActionRule))]
+    public class UpdateAzureActionRule : AlertsManagementBaseCmdlet
+    {
+        #region Parameter Set Names
+
+        private const string ByInputObjectParameterSet = "ByInputObject";
+        private const string ByResourceIdParameterSet = "ByResourceId";
+        private const string ByNameSimplifiedPatchParameterSet = "ByNameSimplifiedPatch";
+
+        #endregion
+
+        #region Parameters declarations
+        /// <summary>
+        /// Gets or sets the resource id parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = ByResourceIdParameterSet, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The resource id of action rule")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the input object
+        /// </summary>
+        [Parameter(ParameterSetName = ByInputObjectParameterSet, Mandatory = true, ValueFromPipeline = true, HelpMessage = "The action rule resource")]
+        public PSActionRule InputObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the action rule name
+        /// </summary>
+        [Parameter(ParameterSetName = ByNameSimplifiedPatchParameterSet, Mandatory = true, HelpMessage = "Action rule name")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource group name
+        /// </summary>
+        [Parameter(ParameterSetName = ByNameSimplifiedPatchParameterSet, Mandatory = true, HelpMessage = "Action rule name")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter]
+        public string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets simplified property of patch object : status
+        /// </summary>
+        [Parameter(ParameterSetName = ByResourceIdParameterSet, Mandatory = false, HelpMessage = "Action rule status")]
+        [Parameter(ParameterSetName = ByInputObjectParameterSet, Mandatory = false, HelpMessage = "Action rule status")]
+        [Parameter(ParameterSetName = ByNameSimplifiedPatchParameterSet, Mandatory = false, HelpMessage = "Action rule status")]
+        [PSArgumentCompleter(ActionRuleStatus.Enabled, ActionRuleStatus.Disabled)]
+        public string Status { get; set; }
```

Comment:
For future reference: this should be split into Enable/Disable cmdlets, since there is no other function of the update cmdlet.